### ISSUE TITLE
How to enter metes and bounds

### ISSUE THREAD

**Issue by zeddock** [mpetroff/qgsazimuth#21]
The help tab in the plugin shows only this:
"Angles may be entered as degrees clockwise from North (nnn.nn or nnd nn' nn.n") or as an offset bearing plus or minus 90 deg. from  North or South (N xxd xx' xx" E)"

I am used to ArcMap but want to move over to QGIS, and this is the area that has me stumped since I do have to create polygons often from scratch, from old deed calls in Appalachia.

Here is a text entry that can be imported into the Traverse tool in ArcMap, just for reference:
SP 1700730.890 323927.150
EP 1696879.220 333721.210
DD N45-00-00W 2590.5
DD S47-00-00W 2277
DD S45-00-00E 2590.5
DD N47-00-00E 2277

The SP and EP are for Starting Point and Ending Point, but that is not critical.
The lines beginning with DD are direction and distance.
The first of them shows a line segment of:
North 45 degrees, 00 minutes, and 00 seconds for a distance of 2590.5 feet.

I am not sure how to input this type of formatting into the plugin. I have tried many different ways but all produce errors.

It looks like the entry, (from the Help tab) should be:
N 45d 00' 00" W

Please help me understand and use this?

Thanx for the development!

![image](https://user-images.githubusercontent.com/26576089/55283844-9c2dcf80-5339-11e9-8442-b8c4cd719cc6.png)

**Comment by mpetroff** [comment]
You're using the wrong input; you need to use the azimuth input.

The zenith angle is for doing vertical corrections, e.g., your next point is on a mountain, so while it's 1000' away line-of-sight, it's 20 deg above the horizon, so it's only 940' away on the map (1000' * cos[20deg]).

**Comment by zeddock** [comment]
Thank you. That helped a bit.
Do you think I am not understanding because of ignorance?

Is it reasonable to be putting N 45D 00' 00" W into the *Azimuth* field?
...and is 90 in Zenith normal?

Thanx!

jim
*     Jim*Garner  *304-933-9858*
*   www.Ground*Scouts.com

**Comment by zeddock** [comment]
I also wanted for you to know that I am seeing the same issues as reported here:
https://gis.stackexchange.com/questions/142408/qgis-azimuth-and-distance-plugin-length-issue

Mine were also resolved.  Although I had Default listed under Options Tab for Distance Unit, I was importing, which likely over-rides most settings.
I changed my top portion of the import file to be:
`angle=Bearing
heading=Coordinate_System
dist_units=Default
angle_unit=degree
startAt=1602437.7904817128;292006.2221059896;90
survey=Polygonal
[data]`
...and it seems to be drawing correctly.

I suspect this is a bug and not just my misunderstanding of the plug-in tool.

All of this said, and for the sake of trying to help, I REALLY appreciate this plugin!

Thank you!

**Comment by mpetroff** [comment]
Yes, `N 45D 00' 00" W` should go in the azimuth field, and the zenith field should be `90` in almost all cases.

The plugin doesn't do proper handling of coordinate systems or units. The problem in the Stack Exchange question you linked to is a result of using a angular coordinate system, WGS84, which isn't supported (nor is a warning displayed). For the plugin to work, you need to use a planar coordinate system such as a State Plane Coordinate System in the United States. As for default / feet, `default` should be used when the distances you're using match the coordinate system's distance unit, e.g., if you're using feet with a State Plane Coordinate System in feet (such as EPSG:2234 for Connecticut). The `feet` selection should only be used if you're using distances in feet with a coordinate system that is in meters (such as EPSG:26956 for Connecticut).

The coordinate system and unit handling could both be improved, as could the documentation. However, I didn't write the plugin and have just been maintaining it enough for it to keep working in newer versions of QGIS (the plugin was originally written for QGIS 1.x).

**Comment by zeddock** [comment]
Well thank you for maintaining it!

Just to be clear, you are saying that the layer into which I am about to
create, is where I want a planar coordinate system, such as:

EPSG:6601 - NAD83(2011) / West Virginia North (ftUS) - Projected?

I hope you continue to help to keep this operational.

Sincerely,

Jim Garner

*     Jim*Garner  *304-933-9858*
*   www.Ground*Scouts.com

**Comment by mpetroff** [comment]
Yes, the target layer needs to have a planar coordinate system.
